types/events.ts

```diff
@@ -47,6 +47,11 @@ export const OnChainEventRequestSchema = z.discriminatedUnion("eventType", [
     bidder: z.string(),
     amount: z.number(),
   }),
+  z.object({
```

Comment:
Should include all fields from above

Reply:
im not sure what you mean by this?

Reply:
I figured `customProgramId` should also be in the zod schema in events.ts here